test/src/org/exist/test/EmbeddedExistTester.java

```diff
@@ -1,140 +0,0 @@
-/*
```

Comment:
@dizzzz After the refactoring it is unused code, so it seems to me there is no point to keep it.